Caddyfile

```diff
@@ -0,0 +1,46 @@
+{
+    auto_https disable_redirects
+
+    storage file_system {
+        root /data
+    }
+
+    log {
+        level ERROR
+    }
+}
+
+(GEOFILTER) {
+        @geofilter {
+                not maxmind_geolocation {
+                        db_path "/usr/share/GeoIP/GeoLite2-Country.mmdb"
+                        allow_countries DE
+                }
+                not remote_ip private_ranges
+        }
+        respond @geofilter 403
+}
+
+https://{$NC_DOMAIN}:443 {
+    import GEOFILTER
+    reverse_proxy nextcloud-aio-apache:{$APACHE_PORT}
+
+    # TLS options
+    tls {
+        issuer acme {
+            disable_http_challenge
+        }
+    }
+}
+
+https://bw.{$NC_DOMAIN}:443 {
+    import GEOFILTER
+    reverse_proxy nextcloud-aio-vaultarden:8812
+
+    # TLS options
+    tls {
+        issuer acme {
+            disable_http_challenge
+        }
+    }
+}
```

Comment:
needs to be aded during start.sh, also only if vaultwarden is there and only import if golite2- is there